protocols/request-response/tests/ping.rs

```diff
@@ -20,31 +20,30 @@
 
 //! Integration tests for the `Behaviour`.
 
-use async_trait::async_trait;
-use futures::{prelude::*, AsyncWriteExt};
-use libp2p_core::upgrade::{read_length_prefixed, write_length_prefixed};
+use futures::prelude::*;
 use libp2p_identity::PeerId;
 use libp2p_request_response as request_response;
 use libp2p_request_response::ProtocolSupport;
 use libp2p_swarm::{StreamProtocol, Swarm, SwarmEvent};
 use libp2p_swarm_test::SwarmExt;
 use rand::{self, Rng};
-use std::{io, iter};
+use serde::{Deserialize, Serialize};
+use std::iter;
 
 #[async_std::test]
 async fn is_response_outbound() {
     let _ = env_logger::try_init();
     let ping = Ping("ping".to_string().into_bytes());
     let offline_peer = PeerId::random();
 
-    let protocols = iter::once((
-        StreamProtocol::new("/ping/1"),
-        request_response::ProtocolSupport::Full,
-    ));
-    let cfg = request_response::Config::default();
-
     let mut swarm1 = Swarm::new_ephemeral(|_| {
-        request_response::Behaviour::with_codec(PingCodec(), protocols, cfg)
+        request_response::cbor::Behaviour::<Ping, Pong>::new(
```

Comment:
A minor thing: This test now depends on the `cbor` feature. To be fully correct, we should specify `required_features` for this test.

Reply:
@thomaseizinger I just added it and to be perfectly consistent, I did the same on the other tests for which the same problem arose.